tty/pc-tty/ttypc_kbd.c

```diff
@@ -16,14 +16,45 @@
 #include <stdint.h>
 #include <string.h>
 #include <unistd.h>
+#include <stdbool.h>
 
+#include <sys/file.h>
 #include <sys/interrupt.h>
 #include <sys/io.h>
 #include <sys/threads.h>
 #include <sys/reboot.h>
 
+#include <libklog.h>
+#include <fcntl.h>
+#include <posix/utils.h>
+
 #include "ttypc_kbd.h"
+#include "ttypc_mouse.h"
 #include "ttypc_vga.h"
+#include "ttypc_ps2.h"
+#include "event_queue.h"
+
+
+#define CTL_THREAD_PRIORITY  1
+#define POOL_THREAD_PRIORITY 1
+
+
+int __attribute__((weak)) ttypc_mouse_handle_event(ttypc_t *ttypc)
```

Comment:
I'm not in favor of this approach. You can provide an alternative implementation of these functions in "ttypc_mouse.c" based on the value of `PC_TTY_MOUSE`.